test/stdlib/Hash_test.rb

```diff
@@ -185,6 +185,11 @@ def test_eql?
     { a: 2 }.eql?({ a: 1 })
   end
 
+  def test_except
+    { a: 100, b: 200, c: 300 }.except(:a)
```

Comment:
Could you add a guard like:

```ruby
skip if Hash.method_defined?(:except)
```

Reply:
https://github.com/ruby/rbs/pull/631/commits/ec6d8651784e5e9acc19aefbd8255a00c7ba0da4

test-unit has `omit_if` feature, does it fit for this case?
Or it is a noisy reporting? 🤔

Reply:
Ah, it looks better! 🙆‍♂️